src/styles/utils/variables.scss

```diff
@@ -0,0 +1,5 @@
+$cell-size: 100px;
+$columns: 7;
+$padding: 10px;
```

Comment:
[Important]

[STYLES] - Don't be greedy, add some top paddings, so the top row don't go over top of the page on hover.

Change padding to 20px 10px to add top paddings tall enough that, when you click a day on the top row, it does not move out of the top part of the screen.